src/openms/source/ANALYSIS/ID/SiriusAdapterAlgorithm.cpp

```diff
@@ -197,7 +197,7 @@ namespace OpenMS
           unsigned int num_masstrace_filter = sirius_algo.filter_by_num_masstraces_;
           double precursor_mz_tol = sirius_algo.precursor_mz_tolerance_;
           double precursor_rt_tol = sirius_algo.precursor_rt_tolerance_;
-          bool ppm_prec = (sirius_algo.precursor_mz_tolerance_unit_ == "true") ? true : false; 
+          bool ppm_prec = (sirius_algo.precursor_mz_tolerance_unit_ == "ppm") ? true : false; 
```

Comment:
```suggestion
          bool ppm_prec;
          if (sirius_algo.precursor_mz_tolerance_unit_ == "ppm") ppm_prec = true;
          else if (sirius_algo.precursor_mz_tolerance_unit_ == "Da") ppm_prec = false;
          throw Exception::IllegalArgument(__FILE__, __LINE__, OPENMS_PRETTY_FUNCTION, "Precursor m/z tolerance unit is either ppm or Da");
```

Reply:
prevent this from happening in the future - either by doing this or by adding regression tests? also for `feature_only_` ?

Reply:
Will add that for feature_only_ as well. 
The test should be fine now since the older ones use Da and the newer (SIRIUS) use ppm.